web-test-runner.config.mjs

```diff
@@ -9,10 +9,10 @@ export default {
   },
   coverageConfig: {
     threshold: {
-      statements: 80,
+      statements: 70,
```

Comment:
This is fine for now, but we need to have more of a conversation about test coverage. We should be adding more tests to cover code rather than lowering the thresholds. By default, all components should be at least at 80%.

Reply:
Maybe open a ticket for this?

Reply:
this change is temporary workaround, recommended by Jason. The ultimate change will be needed ticketed at https://github.com/AlaskaAirlines/auro-form/issues/47